Project/src/main/java/app/EditorController.java

```diff
@@ -0,0 +1,23 @@
+package app;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class EditorController extends app.Controller {
+    @Autowired
+    private FileRepository repository;
```

Comment:
Just move this to the Controller super class. Can you abstract it from the UploadController as well?